biothings/web/handlers/query.py

```diff
@@ -149,9 +149,13 @@ async def get(self):
             meta["software"] = self.biothings.devinfo.get()
 
         else:  # remove debug info
-            for field in list(meta):
-                if field.startswith("_"):
-                    meta.pop(field, None)
+            meta = {
+                "biothing_type": meta.get("biothing_type"),
+                "build_date": meta.get("build_date"),
+                "build_version": meta.get("build_version"),
+                "src": meta.get("src"),
+                "stats": meta.get("stats"),
+            }
```

Comment:
@everaldorodrigo this can be problematic, as each API might have other metadata fields to be included. E.g. MyGene.info/metadata has `genome_assembly`.